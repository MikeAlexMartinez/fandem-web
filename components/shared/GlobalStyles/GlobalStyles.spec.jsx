import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import GlobalStylesComponent from './GlobalStyles';

describe('<GlobalStyles />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<GlobalStylesComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
