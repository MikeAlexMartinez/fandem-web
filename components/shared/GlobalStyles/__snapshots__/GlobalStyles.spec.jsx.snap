// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalStyles /> renders and matches the snapshot 1`] = `
<Page
  classes={Object {}}
/>
`;
